csharp-intermediate-topics/ReadOnlyModifierInCSharp/ReadOnlyModifierInCSharp/Circle.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace ReadOnlyModifierInCSharp
+{
+    public class Circle
+    {
+        public readonly double _radius;
```

Comment:
Naming conventions for a public field is Pascal case:
https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions